src/elements/Pagination.tsx

```diff
@@ -1,7 +1,7 @@
 import { PaginationItem } from '@mui/material';
 import type { Dispatch, SetStateAction } from 'react';
 import React from 'react';
-import * as t from '../../style/pagination.style';
+import * as t from '../style/pagination.style';
 
 function Pagination({ pageCnt, page, setPage }: PropsType) {
```

Comment:
```suggestion
function Pagination({ pageCnt, page, setPage }: PropsType) {
```
컨벤션에 맞지 않는 타입이름이 보이네요! 수정해주시면 좋을 것 같습니다.😊